eos/Makefile.am

```diff
@@ -73,7 +74,8 @@ LDADD = \
 check_PROGRAMS = \
 	constraint_TEST \
 	observable_TEST \
-	reference_TEST
+	reference_TEST \
+	signal-pdf_TEST
```

Comment:
Unrelated to this PR: Why does this list appear twice?

Reply:
Ah, a peculiarity of automake. It has two targets: ``check-PROGRAMS`` and ``check-TESTS``. The former build the list of all programs (``check_PROGRAMS``) that must be available to the tests. The latter run all the tests in ``TESTS``.